packages/directives/resize/index.ts

```diff
@@ -0,0 +1,18 @@
+import { addResizeListener, removeResizeListener } from '@element-plus/utils/resize-event'
+
+import type { ObjectDirective } from 'vue'
+
+function handleResize(el, binding) {
+  el && binding.value?.()
+}
+
+const Resize: ObjectDirective = {
+  beforeMount(el, binding) {
+    addResizeListener(el, handleResize.bind(null, el, binding))
+  },
+  beforeUnmount(el) {
+    removeResizeListener(el, handleResize)
+  },
```

Comment:
The code is simply wrong since the binding function is not the same function compare to the destruction function
`bind` creates a new function, and you remove resize listener with another function.